pkg/agent/cache/manager.go

```diff
@@ -153,11 +158,13 @@ func (m *manager) Init() {
 					if _, ok := m.spiffeIdEntryMap[parentId]; ok {
 						svid = m.spiffeIdEntryMap[parentId].SVID
 						key = m.spiffeIdEntryMap[parentId].PrivateKey
-					}
-					if parentId == m.baseSPIFFEID {
+					} else if parentId == m.baseSPIFFEID || parentId == m.magicSPIFFEID {
 						entry := m.getBaseSVIDEntry()
 						svid = entry.svid
 						key = entry.key
+					} else {
+						m.log.Debug("Unknown parent ", parentId, "... ignoring")
```

Comment:
An info or warn level may be more appropriate, since reaching this point means something is probably wrong. Also, you can use `Infof` or `Warnf` for proper string formatting